integration-cli/docker_api_containers_unix_test.go

```diff
@@ -0,0 +1,91 @@
+// +build !windows
+
+package main
+
+import (
+	"net/http"
+
+	"github.com/docker/docker/integration-cli/checker"
+	"github.com/docker/docker/integration-cli/request"
+	"github.com/docker/docker/pkg/sysinfo"
+	"github.com/go-check/check"
+)
+
+func (s *DockerSuite) TestContainersRunWithHugetlbLimit(c *check.C) {
+	testRequires(c, DaemonIsLinux)
+	testRequires(c, hugetlbLimitSupport)
```

Comment:
Need `testRequires(c, SameHostDaemon)` so the sysinfo check below works as expected.